### PR TITLE
Entry: Semi-Supervised Learning (Mark Muscat)

### PR DESCRIPTION
**Pull request by kernelPilgrim**

### CODE REVIEW

**Comment by kernelPilgrim** [comment]
Uploaded `ics5110_ml_handbook.tex`

File: terms/semisupervised_mm.tex [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+\begin{enumerate}
+\item To predict the labels for the upcoming data instances\index{data instance} in the test set\index{test set}. This is known as \textbf{Inductive Semi-Supervised Learning}\index{inductive semi-supervised learning}.
+\item To predict the labels of the unlabelled\index{unlabelled} data instances\index{data instances} inside of the training set\index{training set}. This is known as \textbf{Transductive Semi-Supervised Learning}\index{transductive semi-supervised learning}.
+\end{enumerate}
+
+Formally, in Inductive Semi-Supervised Learning \citep{zhu2009introduction}\index{inductive semi-supervised learning}, given a training sample; 
+
+\begin{equation}
+\{(\textbf{x}_i, y_i)\}^{l}_{i=1},\{\textbf{x}_j\}^{l+u}_{j=l+1}
```
Comment (jp-um) [inline terms/semisupervised_mm.tex]: Absolutely NO !! `textbf` is used for text and not in equations.  Look at [this entry here](https://tex.stackexchange.com/questions/595/how-can-i-get-bold-math-symbols)

File: terms/semisupervised_mm.tex [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+...our model learns a function $f:X \mapsto Y$, such that $f$ is expected to be a good predictor on future data, beyond $\{\textbf{x}_j\}^{l+u}_{j=l+1}$.
+
+In Transductive Learning \citep{zhu2009introduction}\index{transductive semi-supervised learning}, given a training sample;
+
+\begin{equation}
+\{(\textbf{x}_i,y_i)\}^l_{i=1},\{\textbf{x}_j\}^{l+u}_{j=l+1}
+\end{equation}
+
+...our model learns a function $f: X^{l+u} \mapsto Y^{l+u}$, such that $f$ is expected to be a good predictor on the unlabelled data $\{\textbf{x}_j\}^{l+u}_{j=l+1}$.
```
Comment (jp-um) [inline terms/semisupervised_mm.tex]: Again see guideline 35 in README

File: terms/semisupervised_mm.tex [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+To combat this limitation, among other improvements, one also finds other, more complex algorithms\index{algorithm}. Transductive SVMs\index{transductive SVM} \citep{bennett1999semi} are Support Vector Machines\index{support vector machine} embedded with Semi-Supervised Learning\index{semi-supervised learning}.These methods however have difficulty scaling to large amounts of data. Graph-Based Methods\index{graph-based method} \citep{goldberg2006seeing, zhu2009introduction} are some of the most used techniques\index{semi-supervised technique}. Labelled information\index{labelled information} is spread through the graph from labelled\index{labelled node} to unlabelled nodes\index{unlabelled node}, connecting similar observations. One also finds Neural Network\index{neural network} solutions such as Generative Models\index{generative model} and Deep Generative Models\index{deep generative model} \citep{zhu2009introduction, kingma2014semi}, which are capable of allowing a more robust set of features to be used than linear embedding\index{linear embedding} used by other solutions.
+
+\index{class options|)}
```
Comment (jp-um) [inline terms/semisupervised_mm.tex]: wrong this is a multipage index ... look at https://en.wikibooks.org/wiki/LaTeX/Indexing#Multiple_pages on how to do this.  Also this should be named "Semi-Supervised Learning" and should have a matching start tag,

**Comment by jp-um** [comment]
You are going to open a new PR?

*Dr Jean-Paul Ebejer**,*  BSc (Hons) (Melita), MSc (Imperial), DPhil (Oxon.)

*Lecturer*
Centre for Molecular Medicine and BioBanking,
Room 320, Biomedical Sciences Building,
University of Malta, Msida MSD 2080.
T: (00356) 2340 3263

<https://twitter.com/malteseunderdog>

**Pull request closed without merging**